plugins/integrations/kubernetes-service/src/main/java/com/cloud/kubernetes/cluster/KubernetesClusterManagerImpl.java

```diff
@@ -606,13 +610,20 @@ public KubernetesClusterResponse createKubernetesClusterResponse(long kubernetes
         response.setEndpoint(kubernetesCluster.getEndpoint());
         response.setNetworkId(ntwk.getUuid());
         response.setAssociatedNetworkName(ntwk.getName());
-        List<String> vmIds = new ArrayList<String>();
+        List<UserVmResponse> vmIds = new ArrayList<UserVmResponse>();
```

Comment:
@davidjumani considering that this variable has changed from a list of _VM IDs_ to a list of _VM Responses_ I would suggest renaming it.
What do you think of _vmResponses_ or _vmResponseList_?